test/TestsHelper.js

```diff
@@ -460,18 +460,23 @@ TestsHelper.buildCmd = function buildCmd(baseCmd, positionalArgs, options, flags
 TestsHelper.testTooManyArgs = function testTooManyArgs(baseCmd, additionalArgsCount, done) {
   const additionalArgs = Array(additionalArgsCount).fill('redundantArg');
   const cmd = TestsHelper.buildCmd(baseCmd, additionalArgs);
-  TestsHelper.execCmdWithAssertion(cmd, null, null, true, false, false, (err) => {
+  TestsHelper.execCmdWithAssertion(cmd, null, null, true, false, false, null, (err) => {
     expect(err).to.not.exist;
     done();
   });
 };
 
 TestsHelper.execCmd = function execCmd(cliCmd, options, done) {
-  options = options || {
-    env: {
+  options = options || {};
+  if (options.env) {
+    options.env.PATH = options.env.PATH || process.env.PATH;
+    options.env.NODE_CONFIG = options.env.NODE_CONFIG || JSON.stringify(testsConfig);
+  } else {
+    options.env = {
+      PATH: process.env.PATH,
```

Comment:
I think it should be more like:
- whether or not options are null or not => set `PATH` (because of travis and add comment)
- if options are null => set `NODE_CONFIG`
- if options are not null => do not set `NODE_CONFIG`

And now options may not be null and `NODE_CONFIG` may intentionally be not set but this will set it.